### ISSUE TITLE
 OdooRecords XML File, java.lang.StringIndexOutOfBoundsException

### ISSUE THREAD

**Issue by zalei** [wt-io-it/odoo-pycharm-plugin#63]
Plugin Version: 0.6.2
IntelliJ Version: PyCharm 2022.2.3
IntelliJ Build: PC-222.4345.23

<details><summary>Unhandled exception in [StandaloneCoroutine{Cancelling}@51dda05c, EDT]</summary>

```
java.lang.StringIndexOutOfBoundsException: begin 4, end -1, length 33
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4604)
	at java.base/java.lang.String.substring(String.java:2707)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getOdooRecord(OdooGoToDeclarationHandler.java:88)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTarget(OdooGoToDeclarationHandler.java:72)
	at com.intellij.codeInsight.navigation.actions.GotoDeclarationHandlerBase.getGotoDeclarationTargets(GotoDeclarationHandlerBase.java:14)
	at at.wtioit.intellij.plugins.odoo.OdooGoToDeclarationHandler.getGotoDeclarationTargets(OdooGoToDeclarationHandler.java:33)
	at com.intellij.codeInsight.navigation.impl.GtdProvidersKt.fromGTDProvidersInner(gtdProviders.kt:30)
	...
	at com.intellij.openapi.application.rw.InternalReadAction$runReadAction$4.invoke(InternalReadAction.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:264)
	at com.intellij.openapi.application.rw.InternalReadAction.runReadAction(InternalReadAction.kt:32)
	...
	at com.intellij.codeInsight.navigation.CtrlMouseHandler2$compute$2.invokeSuspend(CtrlMouseHandler.kt:221)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	...
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```
</details>

**Comment by ap-wtioit** [comment]
should be the same as #61, will probably work on a fix on the weekend (@bigbear3001)

**Comment by ap-wtioit** [comment]
this issue is fixed in 0.6.3 (pending jetbrains approval), if you are eager to test it, you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.3

**Comment by ap-wtioit** [comment]
0.6.3 is approved and [now](https://github.com/wt-io-it/odoo-pycharm-plugin/issues/69) also shown for pycharm in marketplace
